docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -346,6 +346,8 @@ disable doc values is by using the `doc_values` property of mappings.
 === Plugin changes
 
 The command `bin/plugin` has been renamed to `bin/elasticsearch-plugin`.
+The structure of the plugin has changed, all the plugin files must be contained into a directory called `elasticsearch`.
```

Comment:
Use a separate sentence (period after "changed", capital A in "All"). Also, into -> in